src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
+
+  // Color
+  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-label--Color: var(--pf-global--Color--light-100);
+
+  // Font
+  --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
+
+  // Modifiers
+  --pf-c-label--m-hover--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-label--m-compact--FontSize: var(--pf-global--FontSize--xs);
+
+  display: inline-block;
+  padding:
+    var(--pf-c-label--PaddingTop)
+    var(--pf-c-label--PaddingRight)
+    var(--pf-c-label--PaddingBottom)
+    var(--pf-c-label--PaddingLeft);
+  font-size: var(--pf-c-label--FontSize);
+  line-height: var(--pf-c-label--LineHeight);
+  color: var(--pf-c-label--Color);
+  text-align: center;
+  white-space: nowrap;
+  background-color: var(--pf-c-label--BackgroundColor);
+  border-radius: var(--pf-c-label--BorderRadius);
+
+  &:hover,
+  &.pf-m-hover {
+    background-color: var(--pf-c-label--m-hover--BackgroundColor);
```

Comment:
can you change the var back to `--pf-c-label--m-hover--BackgroundColor`? the modifier class just supports the existing `hover` state, so we should use the `hover` state in the variable. If the modifier class stood on it's own, then you would use `m-hover` in the var name. You can look at the Button component for some good examples of this.